apps/api/migrations/encrypt-credentials/encrypt-credentials-migration.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { expect } from 'chai';
+import { encryptOldCredentialsMigration } from './encrypt-credentials-migration';
+import { UserSession } from '@novu/testing/src';
+import { ChannelTypeEnum } from '@novu/stateless';
+import { IntegrationRepository } from '@novu/dal/src';
+
+describe('Encrypt Old Credentials', function () {
+  let session: UserSession;
+  const integrationRepository = new IntegrationRepository();
+
+  beforeEach(async () => {
+    session = new UserSession();
+    await session.initialize();
+  });
+
+  it('should decrypt all old credentials', async function () {
+    await pruneIntegration(integrationRepository);
+
+    const data = {
+      providerId: 'sendgrid',
+      channel: ChannelTypeEnum.EMAIL,
+      active: false,
+    };
+
+    for (let i = 0; i < 2; i++) {
+      await integrationRepository.create({
+        _environmentId: session.environment._id,
+        _organizationId: session.organization._id,
+        providerId: data.providerId + i,
+        channel: data.channel,
+        credentials: { user: 'smith' + i, apiKey: '123' + i, secretKey: 'abc' + i },
```

Comment:
Maybe worth adding here other secret fields?

Reply:
Good idea :)

Reply:
I asked on another comment :) , 
does that mean that the other comment addresses the same suggestion as here?